bin/cheminfo-publish.js

```diff
@@ -164,7 +164,16 @@ You chose ${formatToBump(bump)} instead.`);
 
     // Publish package
     console.log('Publishing package');
-    log(yield execNpm('publish'));
+    var publishOutput;
+    try {
+        publishOutput = yield execNpm('publish');
+    } catch(e) {
+        console.error('npm publish failed, rolling back commits and tags');
+        yield child_process.exec(`git tag -d v${newVersion}`);
+        yield child_process.exec('git reset --hard HEAD~1');
```

Comment:
Wouldn't that make you loose commits that you have forgotten to push to origin? You would need to push changes earlier in the script then.

The `reset HEAD~1` can only be executed after the `commit` a little bit higher in the publish script, so I think it is safe